packages/tools/guiEditor/src/components/sceneExplorer/treeItemSelectableComponent.tsx

```diff
@@ -138,7 +147,7 @@ export class TreeItemSelectableComponent extends React.Component<ITreeItemSelect
         const marginStyle = {
             paddingLeft: 10 * (this.props.offset + 0.5) - 20 + "px",
         };
-        const entity = this.props.entity;
+        let entity = this.props.entity;

```

Comment:
just wondering - what is the reason for this change?

Reply:
In the section of code starting at line 230 I change the value of entity, which is why I changed it from const to let. Would it be better practice to create a separate entity variable that I change?

Reply:
no no, this is fine. I just didn't see the line change that re-sets the variable